src/client/app/components/csv/ReadingsCSVUploadComponent.tsx

```diff
@@ -3,338 +3,490 @@
+											<div className='pb-1'>
+												{translate('csv.readings.param.cumulative')}
+											</div>
+											<Input
+												type='select'
+												id='cumulative'
+												name='cumulative'
+												value={getBooleanMeterType(selectedMeter?.cumulative)}
+												onChange={handleChange}
+											>
+												<option value={BooleanMeterTypes.meter}> {translate('BooleanMeterTypes.meter')} </option>
+												<option value={BooleanMeterTypes.true}> {translate('BooleanMeterTypes.true')} </option>
+												<option value={BooleanMeterTypes.false}> {translate('BooleanMeterTypes.false')} </option>
+											</Input>
+										</Label>
+									</FormGroup>
 								</Col>
-							</FormGroup>
-							<FormGroup>
-								<Label style={titleStyle}>
-									<FormattedMessage id='csv.readings.param.cumulative.reset' />
-								</Label>
-								<Col sm={12}>
-									<Input type='select' name='cumulativeReset' onChange={this.handleSetCumulativeReset}>
-										<option value={BooleanMeterTypes.meter}> {translate('BooleanMeterTypes.meter')} </option>
-										<option value={BooleanMeterTypes.true}> {translate('BooleanMeterTypes.true')} </option>
-										<option value={BooleanMeterTypes.false}> {translate('BooleanMeterTypes.false')} </option>
-									</Input>
+								<Col>
+									<FormGroup>
+										<Label for='cumulativeReset'>
+											<div className='pb-1'>
+												{translate('csv.readings.param.cumulative.reset')}
+											</div>
+											<Input
+												type='select'
+												id='cumulativeReset'
+												name='cumulativeReset'
+												value={getBooleanMeterType(selectedMeter?.cumulative)}
+												onChange={handleChange}
+											>
+												<option value={BooleanMeterTypes.meter}> {translate('BooleanMeterTypes.meter')} </option>
+												<option value={BooleanMeterTypes.true}> {translate('BooleanMeterTypes.true')} </option>
+												<option value={BooleanMeterTypes.false}> {translate('BooleanMeterTypes.false')} </option>
+											</Input>
+										</Label>
+									</FormGroup>
 								</Col>
-							</FormGroup>
-							<FormGroup>
-								<Label style={titleStyle}>
-									<FormattedMessage id='csv.readings.param.cumulative.reset.start' />
-								</Label>
-								<Col sm={12}>
-									<Input
-										value={this.props.cumulativeResetStart}
-										name='cumulativeResetStart'
-										onChange={this.handleSetCumulativeResetStart}
-										placeholder={ReadingsCSVUploadDefaults.cumulativeResetStart}
-									/>
+							</Row>
+							<Row>
+								<Col>
+									<FormGroup>
+										<Label for='cumulativeResetStart'>
+											<div className='pb-1'>
+												{translate('csv.readings.param.cumulative.reset.start')}
+											</div>
+											<Input
+												id='cumulativeResetStart'
+												name='cumulativeResetStart'
+												onChange={handleChange}
+												required
```

Comment:
I'm unsure what this required does since I don't see any warning if nothing is entered.